src/edu/csus/ecs/pc2/ui/ResultsComparePane.java

```diff
@@ -0,0 +1,461 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+import java.awt.Font;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+import javax.swing.SwingUtilities;
+import javax.swing.border.TitledBorder;
+
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.FileUtilities;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.execute.ExecuteUtilities;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.model.ClientSettings;
+import edu.csus.ecs.pc2.core.model.Filter;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.report.ExportFilesUtiltiites;
+import edu.csus.ecs.pc2.core.report.ResultsCompareReport;
+import edu.csus.ecs.pc2.exports.ccs.ResultsFile;
+
+/**
+ * Results export and compare pane.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+public class ResultsComparePane extends JPanePlugin {
+
+    private static final String SHOW_COMPARISON_BUTTON_TITLE = "Show Comparison";
+
+    private static final String MISSING_TARGET_TITLE = "Missing from Parimary";
+
+    private static final String MISSING_SOURCE_TITLE = "MIssing from PC2";
```

Comment:
The word "Missing" in the above string should not contain an upper-case "I".  This is not a SHOW STOPPER, but I sure hate to expose things like this to users -- it reflects poorly on the quality of our product.

Reply:
fixed typo